helpers/filterSeats.js

```diff
@@ -86,8 +111,56 @@ module.exports = function(dust) {
       });
     }
 
+    function reorderRates(rates, replyIndex) {
+      var bandTypes = {
+        'gold': [],
+        'silver': [],
+        'bronze': []
+      };
```

Comment:
@sam-cook I can try. The amount of data we pass into this helper and the amount of operations we perform on that data is quite big. Not sure if there is a simple way to mockup the data and check all the operations.  Let me think about it for a while and see what others think might be the best way for testing this. Thanks for reporting. If you have already have an idea on how to do this please let me know too.